build

```diff
@@ -133,7 +133,7 @@ sucrase('./lib', './.lib-dist');
 
 if (sucrase('./server', './.server-dist')) {
 	replace('.server-dist', [
-		{regex: /(require\(.*?)(lib|sim)/g, replace: `$1.$2-dist`},
+		{regex: /(require\(.*?)(data|lib|sim)/g, replace: `$1.$2-dist`},
```

Comment:
o_o

I guess this is for a chat command that directly reads data? Is there a reason `Dex.data` doesn't work?

Reply:
the imported thing we are using from data isn’t accessible via Dex.data